src/lib/Tab/Dashboard/PagerContentToDataMapper.php

```diff
@@ -10,101 +10,51 @@
 
 use eZ\Publish\API\Repository\ContentService;
 use eZ\Publish\API\Repository\ContentTypeService;
+use eZ\Publish\API\Repository\LanguageService;
 use eZ\Publish\API\Repository\UserService;
 use eZ\Publish\Core\Helper\TranslationHelper;
 use eZ\Publish\Core\MVC\Symfony\Locale\UserLanguagePreferenceProviderInterface;
-use EzSystems\EzPlatformAdminUi\Specification\ContentIsUser;
-use EzSystems\EzPlatformAdminUi\Specification\UserExists;
-use Pagerfanta\Pagerfanta;
 
-class PagerContentToDataMapper
+/**
+ * @deprecated This class has been moved to \EzSystems\EzPlatformAdminUi\Search\PagerContentToDataMapper
+ */
+class PagerContentToDataMapper extends \EzSystems\EzPlatformAdminUi\Search\PagerContentToDataMapper
```

Comment:
If I'm not mistaken `\EzSystems\EzPlatformAdminUi\Search\PagerContentToDataMapper` is not used anywhere but on a dashboard tab. If so, please remove `\EzSystems\EzPlatformAdminUi\Search\PagerContentToDataMapper` and extend `AbstractPagerContentToDataMapper` here.

Reply:
This `\EzSystems\EzPlatformAdminUi\Search\PagerContentToDataMapper` is using in dashboard tabs, but the results becomes from Search engines (at least for now). I'm not sure about move AbstractPagerContentToDataMapper to dashboard "domain" because this class are extending also by `\EzSystems\EzPlatformAdminUi\Search\PagerSearchContentToDataMapper`, which is not a part of dashboard "domain", but all three classes are connected by search feature now. Maybe we should move them all to namespace like "Mapper" or "PagerContent" for more independent .. what do you think @webhdx ?